data/css/encyclopedia.scss

```diff
@@ -3,6 +3,7 @@
 $body-font: 'Lato' !default;
 $display-font: 'Lato' !default;
 $title-font: 'Lato' !default;
+$logo-font: 'Libre Baskerville' !default;
```

Comment:
Just a reminder that we need this to be included in eos-theme (or as a dependency).